### DISCUSSION TITLE
javascript\java code for connecting to an smb drive

### DISCUSSION THREAD

**Discussion by rutledj** [nextgenhealthcare/connect#5942] [Q&A]
I need to create a channel to clean out a folder periodically (0 size files and unwanted filenames). I need to connect to a smb drive with a user\pwd. It doesn't appear that javascript provides a way to do this but I'm assuming Java might?

**Comment by daniel-ls-howard** [comment]
Have a look here https://stackoverflow.com/questions/65880643/java-access-network-domain-shared-folder-with-username-and-password

**Comment by rjkroll** [comment]
This thread might also be useful:

https://forums.mirthproject.io/forum/mirth-connect/support/17352-smb-in-javascript-reader
